src/hyperlight_host/src/lib.rs

```diff
@@ -74,7 +74,7 @@ pub use error::HyperlightError;
 /// Re-export for `MemMgrWrapper` type
 /// A sandbox that can call be used to make multiple calls to guest functions,
 /// and otherwise reused multiple times
-pub use sandbox::MultiUseSandbox;
```

Comment:
Maybe we can take this chance to remove these reexports? I'm not sure there is a reason to keep these?